src/pelib/PeLibAux.cpp

```diff
@@ -135,6 +139,23 @@ namespace PeLib
 		return (ByteSize >> PELIB_PAGE_SIZE_SHIFT) + ((ByteSize & (PELIB_PAGE_SIZE - 1)) != 0);
 	}
 
+	bool pelibIsBufferZeroed(const void * buffer, size_t length)
```

Comment:
Is it necessary to prefix the member function with "pelib"? Couldn't we name it just `isBufferZeroed()`?